CPS-XXXX/README.md

```diff
@@ -0,0 +1,160 @@
+---
+CPS: ?
+Title: Full determinism of transactions is unnecessarily restrictive for DeFi
```

Comment:
```suggestion
Title: Relaxed Transaction Determinism
```
@klntsky @Ryun1 please post if you can think of something equally concise that encompasses the proposed changes without bias.

Reply:
I'll come up with something that mentions *mutable shared state*, because this is the term that was used in the eUTxO paper

Reply:
Renamed